### PR TITLE
Pull-Request: Decorate a class

### PR DESCRIPTION
**Pull request by Zven94**
This project use the Decorator design pattern to correct the names of people on my lib

files updated:

- person.rb

### CODE REVIEW

**Review by iambenkis — COMMENTED** [review]
Hi @Zven94  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that does not have a professional README file , Kindly fix that


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,154 @@
...
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors <a name="authors"></a>
+👤 **Nicolas Bolañez**
+- GitHub: [@Nico](https://github.com/Zven9)
+- LinkedIn: [@Nico](https://www.linkedin.com/in/nico-emiliano/)
+
```
Comment (roshan-bajgain) [inline README.md]: - [x] I would suggest you please provide the correct link to you Github page and LinkedIn as it says 404 error.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,154 @@
...
+A: Contributions are highly encouraged! If you have ideas, suggestions, or identify issues, please feel free to submit them through the project's GitHub repository. Refer to the contributing guidelines for more details.
+
+**Q: Where can I find more information about the Ruby programming language?**
+A: For more information about Ruby, its features, and its community, visit the official [Ruby website](https://www.ruby-lang.org/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome! Feel free to check the [issues page](https://github.com/Zven94/OOP-school-library/).
```
Comment (roshan-bajgain) [inline README.md]: - [x] I also suggest you please use the correct link to you issue page as it makes you README more professional. `https://github.com/Zven94/OOP-school-library/issues`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,154 @@
...
+If you find this project helpful, consider giving it a ⭐️ to show your support!
+
+## Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse and all my peers and colleagues at Microverse for giving me the opportunity to work on this project.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (roshan-bajgain) [inline README.md]: - [x] Your MIT license is broken please fix it as it make your README more professional. link `https://github.com/Zven94/OOP-school-library/blob/dev-library/LICENSE`

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Zven94, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- Gitflow is followed. ✔️
-  Implement decorator ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Zven94, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Edited ReadMe File As Requested :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**